website/source/docs/providers/datadog/r/user.html.markdown

```diff
@@ -27,7 +27,7 @@ The following arguments are supported:
 
 * `disabled` - (Optional) Whether the user is disabled
 * `email` - (Required) Email address for user
-* `handle` - (Required) Datadog ID for user (generally same as email)
+* `handle` - (Required) The user handle, must be a valid email.
```

Comment:
Nit; "must be a valid email address". After all we are interested in an *address*, not some email random email 😉